perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -465,9 +467,13 @@ def os_info(self):
     if self.os_metadata.get('os_info'):
       return self.os_metadata['os_info']
     else:
-      self.Install('lsb_release')
-      stdout, _ = self.RemoteCommand('lsb_release -d')
-      return regex_util.ExtractGroup(LSB_DESCRIPTION_REGEXP, stdout)
+      if (self.OS_TYPE == 'clear'):
+        stdout, _ = self.RemoteCommand('swupd info | grep Installed')
```

Comment:
Let's call a method GetOsInfo that installs and runs lsb_release. Then we can override this in the Mixin class for Linux Clear.